rclpy/test/test_logging.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import rclpy
+from rclpy.logging import LoggingSeverity
+
+
+class TestLogging(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        rclpy.logging.initialize()
+
+    def test_set_get_severity_threshold(self):
```

Comment:
Nitpick about the name: Use `*_get_set_*` (since that is the order used in the test) or just `test_severity_threshold`.